internal/pkg/plugin/gitlabcedocker/create.go

```diff
@@ -41,11 +42,15 @@ func Create(options map[string]interface{}) (map[string]interface{}, error) {
 		return nil, fmt.Errorf("failed to run container")
 	}
 
+	// 4. check if the volume is created successfully
 	volumes, err := op.ListContainerMounts(gitlabContainerName)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get container mounts: %v", err)
 	}
 
+	// 5. show the access url
+	showGitLabUrl(opts)
```

Comment:
To be consistent with the rest of the codebase, it should be `showGitLabURL` instead.

Reply:
> Is it necessary to remove `deletion_test.go`?

I think `deletion_test.go` doesn't work properly now.

Reply:
> To be consistent with the rest of the codebase, it should be `showGitLabURL` instead.

Thank you!

Reply:
> > Is it necessary to remove `deletion_test.go`?
> 
> I think `deletion_test.go` doesn't work properly now.

If you think it's not working, why not improve it instead?:relaxed: A well-designed test case can serve as extra documentation for the codebase.

Reply:
Thank you very much, I agree. The test I removed was actually only e2e test for most of the content. But inspired by you, I added as much as I could to the wish test, thanks!